test/e2e/e2e.go

```diff
@@ -1266,6 +1268,154 @@ spec:
 			framework.Failf("Failed to curl the remote host %s from container %s on node %s: %v", exFWPermitTcpWwwDest, ovnContainer, serverNodeInfo.name, err)
 		}
 	})
+	ginkgo.It("Should validate the egress firewall policy functionality against cluster nodes by using node selector", func() {
+		srcPodName := "e2e-egress-fw-src-pod"
+		frameworkNsFlag := fmt.Sprintf("--namespace=%s", f.Namespace.Name)
+		testContainer := fmt.Sprintf("%s-container", srcPodName)
+		testContainerFlag := fmt.Sprintf("--container=%s", testContainer)
+		// use random labels in case test runs again since it's a pain to remove the label from the node
+		labelMatch := randStr(5)
+		// egress firewall crd yaml configuration
+		var egressFirewallConfig = fmt.Sprintf(`kind: EgressFirewall
+apiVersion: k8s.ovn.org/v1
+metadata:
+  name: default
+  namespace: %s
+spec:
+  egress:
+  - type: Allow
+    to:
+      cidrSelector: %s/%s
+  - type: Allow
+    to:
+      cidrSelector: %s
+    ports:
+      - protocol: TCP
+        port: 80
+  - type: Allow
+    to:
+      nodeSelector:
+        matchLabels:
+          %s: %s
+  - type: Deny
+    to:
+      cidrSelector: %s
+`, f.Namespace.Name, exFWPermitTcpDnsDest, singleIPMask, exFWPermitCIDR, f.Namespace.Name, labelMatch, exFWDenyCIDR)
+		framework.Logf("Egress Firewall CR generated: %s", egressFirewallConfig)
```

Comment:
do we need this log?

Reply:
I found it was useful because then I can see the labelMatch string generated

Reply:
that is more of a test debugging right? once you know the CR is correct, you don't need the log?